src/main/java/org/fundacionjala/coding/dennis/SpinString.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.StringJoiner;
+
+/**
+ *
+ */
+public final class SpinString {
+
+    public static final int LIMIT = 5;
+
+    /**
+     *
+     */
+    private SpinString() {
+    }
+
+    /**
+     *
+     * @param st cadena
+     * @return cadena
+     */
+    public static String spinningString(final String st) {
+        StringJoiner sJoiner = new StringJoiner(" ");
+        String[] words = st.split(" ");
+        for (String word : words) {
+            sJoiner.add(word.length() >= LIMIT ? spinBuilder(word) : word);
+        }
+        return sJoiner.toString();
+    }
+
+    /**
+     *
+     * @param word cadena
+     * @return cadena
+     */
+    public static String spinBuilder(final String word) {
```

Comment:
this method is redundant
You can invert the word directly in the previous method.